quantization/existing-data.ipynb

```diff
@@ -0,0 +1,199 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Atlas Vector Search - Vector Quantization - Existing Data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook is a companion for the [Vector Quantization](https://www.mongodb.com/docs/atlas/atlas-vector-search/vector-quantization/#how-to-ingest-pre-quantized-vectors) page. Refer to this page for set up steps and explanation details.\n",
+    "\n",
+    "This notebook takes you through how to pre-quantize and ingest your vectors for vector search from **existing data in Atlas** by using [Cohere's](https://cohere.com/) `embed-english-v3.0` model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip --quiet --upgrade install pymongo cohere"
```

Comment:
```suggestion
    "pip install --quiet --upgrade pymongo cohere"
```
Same as below, I get a "no such command" error unless I move these arguments to after the `install` command.